src/main/java/core/basesyntax/FibonacciSequence.java

```diff
@@ -6,6 +6,19 @@
      * из `n`чисел  последовательности Фибоначчи.
      */
     public int[] getFibonacciSequence(int n) {
-        return null;
+        if (n == 0) {
+            return new int[0];
+        } else  if (n == 1) {
+            int[] arrayFib = {0};
+            return arrayFib;
```

Comment:
you may simplify this. something
```suggestion
            return {0};
```